pom.xml

```diff
@@ -45,7 +45,7 @@
 
     <properties>
         <site.jira.version.id>12315369</site.jira.version.id>
```

Comment:
please add a timestamp property for reproducible builds: https://maven.apache.org/guides/mini/guide-reproducible-builds.html#how-do-i-configure-my-maven-build

Reply:
+1, done